.eslintrc.js

```diff
@@ -7,4 +7,8 @@ module.exports = {
 	rules: {
 		'sort-keys-fix/sort-keys-fix': ['error', 'asc', { caseSensitive: true, natural: true }],
 	},
+	env: {
+		browser: true,
```

Comment:
There are environment specific override eslint config files in the frontend and the server. Can you shift the environment declarations to each of those, respectively?